### PR TITLE
Update java11_fat for Java 16 testing

### PR DESCRIPTION
**Pull request by gjwatts**
Adds a basic Java 16 test to verify we are running with the proper version of Java.

### CODE REVIEW

File: dev/com.ibm.ws.java11_fat/fat/src/com/ibm/ws/java11_fat/Java16Test.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    private static final String APP_NAME = "java16-app";
+
+    @Server("java11_fat-java16-server")
+    public static LibertyServer server;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        // NOTE: This FAT uses a pre-compiled application which is compiled at the bytecode
+        // level of JDK 16, which is higher than what our build systems normally use
+        // Source for the app can be found here: https://github.com/gjwatts/liberty-misc/tree/java16-app/src/main/java
```
Comment (KyleAure) [inline dev/com.ibm.ws.java11_fat/fat/src/com/ibm/ws/java11_fat/Java16Test.java:42]: Looks like this branch does not exist

File: dev/com.ibm.ws.java11_fat/fat/src/com/ibm/ws/java11_fat/Java16Test.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    @AfterClass
+    public static void tearDown() throws Exception {
+//        server.stopServer("CWWJP9991W"); // warnings for JPA's auto drop/create tables
+        server.stopServer();
+    }
+
+    @Test
+    public void testJava16App() throws Exception {
+        String appResponse = HttpUtils.getHttpResponseAsString(server, APP_NAME + '/');
+        assertContains(appResponse, "<<< EXIT SUCCESSFUL");
```
Comment (KyleAure) [inline dev/com.ibm.ws.java11_fat/fat/src/com/ibm/ws/java11_fat/Java16Test.java:56]: If this assertion fails and you are getting a 404 at this endpoint likely the issue is that an exception was thrown. See comments on app on the best way to debug this.

File: dev/com.ibm.ws.java11_fat/src-reference/java16/org/gjwatts/liberty/TestService.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    @GET
+    public String test() {
+        try {
+            log(">>> ENTER");
+            doTest();
+            log("<<< EXIT SUCCESSFUL");
+        } catch (Exception e) {
+            e.printStackTrace(System.out);
+            e.printStackTrace(new PrintWriter(sb));
+            log("<<< EXIT FAILED");
```
Comment (KyleAure) [inline dev/com.ibm.ws.java11_fat/src-reference/java16/org/gjwatts/liberty/TestService.java:24]: You are likely going down this path.  You are printing the stack trace of the exception to system.out and to the response header of the endpoint.
I would take a look at your server logs for the exception being thrown here and debug that.

File: dev/com.ibm.ws.java11_fat/src-reference/java16/org/gjwatts/liberty/TestService.java [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+        }
+        String result = sb.toString();
+        sb = new StringWriter();
+        return result;
```
Comment (KyleAure) [inline dev/com.ibm.ws.java11_fat/src-reference/java16/org/gjwatts/liberty/TestService.java:28]: The mystery to me is why this endpoint is returning 404.  Exception should have been handled and a result returned.  There might be something going on here.

**Review by KyleAure — APPROVED** [review]

**Review by KyleAure — CHANGES_REQUESTED** [review]
Check out my comments and let me know if you need more help debugging this.

File: dev/com.ibm.ws.java11_fat/fat/src/com/ibm/ws/java11_fat/Java16Test.java (hunk 1 above)
Comment (gjwatts) [reply dev/com.ibm.ws.java11_fat/fat/src/com/ibm/ws/java11_fat/Java16Test.java:42]: Fixed, the source is out there now.

**Comment by gjwatts** [comment]
#build

File: dev/com.ibm.ws.java11_fat/src-reference/java16/org/gjwatts/liberty/RecordTest.java [hunk 5]
```diff
@@ -0,0 +1,14 @@
+package org.gjwatts.liberty;
+
+/**
+ * @author GregoryWatts
```
Comment (KyleAure) [inline dev/com.ibm.ws.java11_fat/src-reference/java16/org/gjwatts/liberty/RecordTest.java:4]: I would add copyright to this file and remove author tag.

**Review by KyleAure — APPROVED** [review]

File: dev/com.ibm.ws.java11_fat/src-reference/java16/org/gjwatts/liberty/RecordTest.java (hunk 5 above)
Comment (gjwatts) [reply dev/com.ibm.ws.java11_fat/src-reference/java16/org/gjwatts/liberty/RecordTest.java]: Done.  Thanks, good catch!

**Comment by gjwatts** [comment]
Force pushed the copyright header info and comment.  No code changes made.

**Comment by gjwatts** [comment]
#libbybot

**Comment by gjwatts** [comment]
Force push after rebase due to build instability.

**Comment by gjwatts** [comment]
#build

**Comment by gjwatts** [comment]
#libbybot

**Pull request merged**